packages/action-group/test/action-group.test.ts

```diff
@@ -269,7 +265,7 @@ describe('ActionGroup', () => {
 
         await elementUpdated(el);
 
-        await aTimeout(500);
+        await aTimeout(100);
```

Comment:
At some point we really need to come up with something less brittle than `aTimeout` and `nextFrame`. There has to be a `waitUntil` condition we can use instead.